### PR TITLE
Embedding CSP: Finding effective directive for a given name

### PR DESCRIPTION
**Pull request by Sun77789**
@mikewest

### CODE REVIEW

**Comment by mikewest** [comment]
How do you plan on using this?

File: embedded/index.src.html [hunk 1]
```diff
@@ -636,6 +636,112 @@ <h4 id="subsume-source-list" algorithm>
...
+      ::
+
+      : "`script-src`"
+      ::
+
+      : "`style-src`"
+      ::
+        1.  If |policy|'s <a>directive set</a> contains a
+            <a>directive</a> whose <a for="directive">name</a> is |name|, 
+            return that <a>directive</a>.
```
Comment (Sun77789) [inline embedded/index.src.html]: This assumes there is only one directive for a directive name in a policy. Should we add here if there are multiple, then we take the first one?

**Comment by Sun77789** [comment]
I was thinking of using it here: https://github.com/w3c/webappsec-csp/pull/154/files

**Comment by Sun77789** [comment]
Rebased on https://github.com/w3c/webappsec-csp/pull/154

**Pull request merged**